### PR TITLE
Link to the Cur Conv help

### PR DESCRIPTION
**Pull request by artemkovalyov**
## What Has Changed?

Link to CC page on SAP Help

### CODE REVIEW

File: docs/java/extensions/currency-conversion.mdx [hunk 1]
```diff
@@ -351,3 +351,7 @@ Conversion requests fail due to several reasons, including but not limited to th
 You can handle the individual status of a success or failure exception.
 To learn more, see [Code Example](#code-example).
 :::
+
+## Where to look next
```
Comment (artemkovalyov) [inline docs/java/extensions/currency-conversion.mdx:355]:
```suggestion
## Where To Look Next
```

File: docs/java/extensions/currency-conversion.mdx [hunk 2]
```diff
@@ -351,3 +351,7 @@ Conversion requests fail due to several reasons, including but not limited to th
 You can handle the individual status of a success or failure exception.
 To learn more, see [Code Example](#code-example).
 :::
+
+## Where To Look Next
+
+The SAP Currency Conversion library [documentation](https://help.sap.com/viewer/7080b5b59e8243ecb48e34c1b107538b/Latest/en-US) on the SAP Help Portal
```
Comment (marikaner) [inline docs/java/extensions/currency-conversion.mdx:357]:
```suggestion
Find the SAP Currency Conversion library [API documentation](https://help.sap.com/viewer/7080b5b59e8243ecb48e34c1b107538b/Latest/en-US) on the SAP Help Portal.
```

**Review by marikaner — APPROVED** [review]
I requested a minor change. Other than that LGTM.

**Comment by artemkovalyov** [comment]
Good proposal @marikaner, I used it.

**Pull request merged**